src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14180,19 +14331,19 @@ static void Main()
                 			native int 'method'
                 		) runtime managed 
                 	{
-                	} // end of method '<>f__AnonymousDelegate0'::.ctor
+                	} // end of method '<>f__AnonymousDelegate0`2'::.ctor
                 	.method public hidebysig newslot virtual 
                 		instance void Invoke (
-                			int32 arg1,
-                			int32[] arg2
+                			!T1 arg1,
+                			!T2 arg2
                 		) runtime managed 
                 	{
                 		.param [2]
                 			.custom instance void [{{s_libPrefix}}]System.ParamArrayAttribute::.ctor() = (
                 				01 00 00 00
                 			)
```

Comment:
Note: here a delegate `<>f_AnonymousDelegate0<T1,T2>(T1 arg1, params T2 arg2)` is generated, i.e., `ParamArrayAttribute` is applied to generic parameter `T2` which in theory might not be an array. But in practice it will always be an array because the anonymous delegate is only used for delegates with matching `params` modifier.